server/middleware/verifyToken.js

```diff
@@ -0,0 +1,57 @@
+import jwt from 'jsonwebtoken';
+import dotenv from 'dotenv';
+import { errorRes } from '../utils/responseHandler';
+
+dotenv.config();
```

Comment:
this line should only be called in your app entry file

Reply:
I've removed it